app/views/podcast_metrics/_main_card.html.erb

```diff
@@ -0,0 +1,30 @@
+<%= turbo_frame_tag "main_card" do %>
```

Comment:
this `main_card` was the goal for this PR. the `card-header` houses the tabs and on the right-hand side will display/hide a dropdown specific to whichever card is displayed at the moment.